tests/d3d12_sm_advanced.c

```diff
@@ -4007,7 +4007,7 @@ void test_sm68_draw_parameters(void)
     D3D12_ROOT_SIGNATURE_DESC rs_desc;
     D3D12_RESOURCE_DESC resource_desc;
     ID3D12Resource *uav, *draw_buffer;
-    ID3D12DescriptorHeap *uav_heap;
+    ID3D12DescriptorHeap *uav_heap, *uav_cpu_heap;
```

Comment:
Move the line up further so that lines are sorted by length, (aka, inverse christmas tree).

Reply:
Fixed, thanks!